### PR TITLE
sudoers cleanup; systemd service cleanup

### PR DESCRIPTION
**Pull request by aelkiss**
- Expands sudo access & moves to sudoers.d drop in (#39, #58, #25)
- Removes confusing/old/duplicatives rails-prod role
- Makes systemd services consistent w/ nectar (https://tools.lib.umich.edu/jira/browse/AEID-57)

Not tested beyond doing `ansible-playbook playbook.predeploy.yml --check`

### CODE REVIEW

File: roles/puma-svc/templates/sudoers.d.j2 [hunk 1]
```diff
@@ -0,0 +1,5 @@
+%{{puma_svc_app_group}} ALL=(root) NOPASSWD: /bin/systemctl stop app-puma@{{puma_svc_app_name}}.service"
+%{{puma_svc_app_group}} ALL=(root) NOPASSWD: /bin/systemctl start app-puma@{{puma_svc_app_name}}.service"
+%{{puma_svc_app_group}} ALL=(root) NOPASSWD: /bin/systemctl restart app-puma@{{puma_svc_app_name}}.service"
+%{{puma_svc_app_group}} ALL=(root) NOPASSWD: /bin/systemctl status app-puma@{{puma_svc_app_name}}.service"
+%{{puma_svc_app_group}} ALL=(root) NOPASSWD: /bin/journalctl
```
Comment (botimer) [inline roles/puma-svc/templates/sudoers.d.j2:5]: Is journalctl supposed to be specific to the service here?

**Review by botimer — COMMENTED** [review]
My understanding is that all the rails-prod stuff was decomposed into db, db-user, etc. Is that right? With confirmation on that and the `journalctl` entry, this looks good to merge.

File: roles/puma-svc/templates/sudoers.d.j2 (hunk 1 above)
Comment (aelkiss) [reply roles/puma-svc/templates/sudoers.d.j2:5]: No, we were intending broad journalctl access (we have been granting that fairly widely). I'm sure there are some dangerous options there, but generally we trust our devs not to intentionally falsify or purge logs...

**Comment by aelkiss** [comment]
The rails-prod role wasn't being used anywhere, and it certainly looks like it had been decomposed. I'm not sure why it was still hanging around in there.

**Pull request merged**